packages/beacon-node/src/network/reqresp/handlers/lightClientBootstrap.ts

```diff
@@ -0,0 +1,15 @@
+import {altair} from "@lodestar/types";
+import {IBeaconChain} from "../../../chain/index.js";
+import {ResponseError} from "../response/index.js";
+import {RespStatus} from "../../../constants/index.js";
+
+export async function* onLightclientBootstrap(
+  requestBody: altair.BlockRoot,
+  chain: IBeaconChain
+): AsyncIterable<altair.LightClientBootstrap> {
+  try {
+    yield await chain.lightClientServer.getBootstrap(requestBody);
+  } catch (e) {
+    throw new ResponseError(RespStatus.RESOURCE_UNAVAILABLE, (e as Error).message);
```

Comment:
Note: The `chain.lightClientServer.getBootstrap` could fail to produce a result for various reasons...this, according to the spec, should lead to an error code 3: ResourceUnavailable. The `(e as Error).message` provides additional textual information what error it was

Reply:
This is not okay, getBootstrap may throw for other reasons that are not RESOURCE_UNAVAILABLE

Reply:
> it must return ServerError, check the spec

The spec here https://github.com/ethereum/consensus-specs/blob/dev/specs/altair/light-client/p2p-interface.md#getlightclientbootstrap which I am mainly following for the implementation says nothing about ServerError. Please do share the part of the spec you are referring to.